iiblib/iib_build_details_model.py

```diff
@@ -1,135 +1,160 @@
 class IIBBuildDetailsModel(object):
     """Model class handling data about index build task"""
 
-    def __init__(
-        self,
-        _id,
-        state,
-        reason,
-        state_history,
-        from_index,
-        from_index_resolved,
-        bundles,
-        removed_operators,
-        organization,
-        binary_image,
-        binary_image_resolved,
-        index_image,
-        request_type,
-        arches,
-        bundle_mapping,
-        omps_operator_version,
-    ):
-        """
-        Args:
```

Comment:
Can you please move this documentation of attributes to class level?

Reply:
moved